### DISCUSSION TITLE
Octernship Query

### DISCUSSION THREAD

**Discussion by Monson2002** [community/community#66258] [GitHub Education]
### Hi everyone,

Can I know when new Octernship are arriving, because its been a long time since I last saw an Octernship listed here

**Comment by arafajs** [comment]
@Monson2002 
The GitHub Education team has not announced a specific date for the next round of Octernships. However, based on the past, new Octernships are usually announced in the fall. So, it's possible that we'll see new Octernships being offered sometime in September or October of this year.

In the meantime, you can stay up-to-date on the latest news about Octernships by following the GitHub Education Twitter account (@GithubEducation). They often announce new Octernships and other opportunities on Twitter.

**Comment by 20481A05K0** [comment]
Hey @Monson2002 I am also looking for the same thing yaar. If anybody know please intimate to us 👍

**Comment by sharma62** [comment]
Hy @Monson2002 and other folks if you have any information then spread with us. I regularly checked it.

**Comment by 20481A05K0** [comment]
Respond yaar

**Comment by 0xatulpatil** [comment]
#66555

**Comment by 20481A05K0** [comment]
Buy please acknowledged when the dates are announced and one more query is that does anyone apply for MLH fellowship for spring 2024

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
